test/test_torch.py

```diff
@@ -1653,14 +1655,6 @@ def reference(x, k, o3, o32):
         self._test_conv_corr_eq(lambda x, k: torch.xcorr3(x, k), reference)
 
     @unittest.skip("Not implemented yet")
-    def test_xcorr3_xcorr2_eq(self):
```

Comment:
Oh I see. The function to be tested doesn't even exist yet. What should the two different functions be called?